examples/simulated_edgesystem_graphite.py

```diff
@@ -55,16 +55,17 @@ def simulated_sampling_function():
 
 if __name__ == '__main__':
 
-    system = SimulatedSystem(config['SystemName'])
+    edgesystem = SimulatedEdgeSystem(config['EdgeSystemName'])
```

Comment:
If **EdgeSystem** is the camel case, then the corresponding variable name should be **edge_system**, saying they both contain two words.